worker-message-prioritization-rabbitmq/src/main/java/com/github/workerframework/workermessageprioritization/rabbitmq/ShovelsApi.java

```diff
@@ -27,9 +27,12 @@ public interface ShovelsApi {
     @PUT("/api/parameters/shovel/{vhost}/{name}")
     RetrievedShovel putShovel(@Path("vhost") final String vhost, @Path("name") final String name, @Body final Component<Shovel> shovel);
     
-    @GET("/api/shovels/")
+    @GET("/api/shovels/") // Used by the 'Shovel Status' UI: https://host/#/shovels
```

Comment:
I'm not sure, when these shovels were on moe, recreating the shovel had no effect (i.e doing another PUT), but deleting the shovel worked, and allowed the shovel to be recreated during the next loop, so that's why I went with deleting the shovel here. 

Unfortunately, the shovels are no longer on moe, so I cannot test if restarting them will work.

I could try and restart them here before deleting them, but I think it's risky as I am not sure what the outcome will be, e.g., maybe the response from the restart API call is successful, but the shovels are still in this stuck state. By deleting them, at least I know that will get things moving again.

Reply:
Regarding the vhost, I have updated the /api/shovels endpoint to use a vhost as well, to keep it consistent with the other endpoints:

https://github.com/WorkerFramework/worker-message-prioritization/pull/25/commits/8b53816e953643bef440569bf40220b898310c15